### DISCUSSION TITLE
Prusa or Orca Slicer Start G Code Example

### DISCUSSION THREAD

**Discussion by Rneil-ca** [TypQxQ/KTC#45] [Configuring] [answered]
Hi ,

Sorry for another new discussion, but I am so close to get this all working.  Thank you again for building this, it is great.  Do you have an example of what should be in PrusiaSlicer or Orcaslicer start G-code?  I though I found an example somewhere when I was digging through the config examples and issues, but I look everywhere again and could not find this.

Thanks again,
Ryan

**Comment by TypQxQ** [comment, marked as answer]
Don't know why it disapeared but I added it again:
[config/PrusaSlicer_Custom_Gcode.md](https://github.com/TypQxQ/KTC/blob/main/config/PrusaSlicer_Custom_Gcode.md)

**Reply by Rneil-ca** [reply]
Thank you, that is what I was looking for!
